Fix calendar glitches

taahol opened the pull request:
Näköjään koodissa ollut off-by-one-korjauspurkka ei olekaan tarpeen. Säädetty myös viivamerkkien typografiaa.

kovipu commented on components/calendar.tsx:
```diff
@@ -102,20 +102,20 @@ const Event: FC<{ event: Event }> = ({ event }) => {
-  // subtract one, as the end date is always set to midnight causing an off-by-one
-  end.setDate(end.getDate() - 1);
+  // FIXME: is there possible off-by-one errors?
+  end.setDate(end.getDate());
```
Hyvä fiksi! Tää rivi ei näytä tekevän mitään ilman tota `- 1`, joten voit vaikka poistaa koko rivin?

taahol replied:
No joo, totta turiset.

taahol committed:
Fix calendar glitches

Näköjään koodissa ollut off-by-one-korjauspurkka ei olekaan
tarpeen. Säädetty myös viivamerkkien typografiaa.
